bookkeeper-server/src/main/java/org/apache/bookkeeper/client/impl/BookKeeperBuilder.java

```diff
@@ -18,10 +18,13 @@
  * under the License.
  *
  */
-package org.apache.bookkeeper.client.api;
+package org.apache.bookkeeper.client.impl;
```

Comment:
hmm. I wasn't suggesting moving interface to impl package. I was suggesting moving BookKeeperBuilderImpl to impl package. the BookKeeperBuilder interface should still in `api` package.

Reply:
sorry I moved the wrong class by mistake